meta/main.yml

```diff
@@ -1,20 +1,11 @@
 ---
 galaxy_info:
-  author: Paul Durivage
-  description: Docker on Ubuntu greater than 12.04
+  author: Location Labs
+  description: Docker on Ubuntu >= 12.04
   license: Apache v2.0
-  min_ansible_version: 1.2
+  min_ansible_version: 1.9.6
   platforms:
     - name: Ubuntu
-      versions:
```

Comment:
It's YAML, which [supports comments](http://www.yaml.org/spec/1.2/spec.html#id2780069).